client/src/lib/firebase/firestore-functions.ts

```diff
@@ -1,9 +1,12 @@
 import { functions } from "$lib/firebase/app";
 import { httpsCallable } from "firebase/functions";
-import type { LobbyRequest } from "./firestore-functions-types";
+import type { LobbyRequest, ChatRequest, StalkChatroomRequest } from "./firestore-functions-types";
 import type { ChangeAvatarData } from "./functions-types/avatar";
 
 export const startGame = httpsCallable<LobbyRequest, void>(functions, "startGame");
 export const joinLobby = httpsCallable<LobbyRequest, void>(functions, "joinLobby");
+export const stalkChatroom = httpsCallable<StalkChatroomRequest, void>(functions, "stalkChatroom");
+export const deleteChatRooms = httpsCallable<LobbyRequest, void>(functions, "deleteChatRooms");
```

Comment:
this shouldn't be here. Are you merged with development?

Reply:
i am merged with dev, where should it be?

Reply:
Delete line 9